public/v4/apps/mattermost-ee.yml

```diff
@@ -0,0 +1,53 @@
+captainVersion: 4
+services:
+    $$cap_appname-db:
+        image: postgres:$$cap_pg_version
+        volumes:
+            - $$cap_appname-db-data:/var/lib/postgresql/data
+        restart: always
+        environment:
+            POSTGRES_USER: mmuser
+            POSTGRES_PASSWORD: $$cap_pg_pass
+            POSTGRES_DB: mattermost
+        caproverExtra:
+            notExposeAsWebApp: 'true'
+    $$cap_appname:
+        image: mattermost/mattermost-enterprise-edition:$$cap_mattermost_version
+        depends_on:
+            - $$cap_appname-db
+        volumes:
+            - $$cap_appname-data:/mattermost/data
+            - $$cap_appname-logs:/mattermost/logs
+            - $$cap_appname-config:/mattermost/config
+            - $$cap_appname-plugins:/mattermost/plugins
+            - $$cap_appname-clientplugins:/mattermost/client/plugins
+        restart: always
+        environment:
+            MM_USERNAME: mmuser
+            MM_PASSWORD: $$cap_pg_pass
+            MM_DBNAME: mattermost
+            DB_HOST: srv-captain--$$cap_appname-db
+        caproverExtra:
+            containerHttpPort: '8065'
+caproverOneClickApp:
+    variables:
+      - id: $$cap_pg_version
+        label: postgres Docker Image tag
+        defaultValue: 13.1
+        description: Check out their Docker page for the valid tags https://hub.docker.com/_/postgres
+      - id: $$cap_mattermost_version
+        label: mattermost Docker Image tag
+        defaultValue: v5.31.1
```

Comment:
This is not a tag for this image: https://hub.docker.com/r/mattermost/mattermost-enterprise-edition/tags?page=1&ordering=last_updated

Did you mean to use mattermost-enterprise-edition or mattermost-prod-app?

Reply:
thanks,
fixed